chainerx_cc/CMakeLists.txt

```diff
@@ -179,6 +183,12 @@ get_third_party(optional-lite)
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/optional-lite/include)
 
 # Test
+
+# ChainerX is linked with MultiThreaded DLL. So I specify gtest with MultiThreaded DLL.
```

Comment:
A wording suggestion. Let's avoid ambiguous pronoun ("I").

```suggestion
# ChainerX is linked with MultiThreaded DLL, including gtest.
```

Reply:
I fixed it